tt-metal-yolov4/README.md

```diff
@@ -0,0 +1,41 @@
+# TT Metalium YoloV4 Inference API
+
+This implementation supports YoloV4 execution on Grayskull and Worhmole.
```

Comment:
```suggestion
This implementation supports YoloV4 execution on Grayskull and Wormhole.
```

Reply:
Changed in https://github.com/tenstorrent/tt-inference-server/pull/52/commits/4c0a27738c78837bea449291e75330d4b3cdebed